packages/web/docs/src/pages/docs/gateway/deployment/runtimes/bun.mdx

```diff
@@ -13,7 +13,16 @@ platform besides Node.js. [Bun](https://bun.sh) is a modern JavaScript runtime l
 and it supports Fetch API as a first class citizen. So the configuration is really simple like any
 other JS runtime with Hive Gateway;
 
-The following code is a simple example of how to use Hive Gateway with Bun.
+## Hive Gateway CLI
+
+You can follow the introduction page directly to [use Hive Gateway CLI](/docs/gateway#installation).
+
+Since Bun has the compatibility layer for Node.js, all Node specific features are available in Bun.
```

Comment:
That's kinda correct, but I think we don't have a decision to promote Bun as the recommended runtime?

Reply:
This is Bun's page, so we don't recommend it but we say you can use Hive Gateway CLI with Bun. Previously it was just runtime.